lighthouse-core/scripts/update-flow-fixtures.js

```diff
@@ -70,6 +70,8 @@ async function waitForImagesToLoad(page) {
 
     await flow.navigate('https://www.mikescerealshack.co/corrections');
 
+    await flow.endFlow();
+
     const flowResult = flow.getFlowResult();
```

Comment:
is there a reason to have separate functions for endFlow and getFlowResult?

Reply:
I didn't want to make `getFlowResult` async but that concern is kinda thin. I'm up for either.